ansible/roles/dns-dhcp/tasks/main.yml

```diff
@@ -9,10 +9,17 @@
 #  the blacklisting isn't applied immediately)
 - name: Blacklist 127.0.0.1 as a nameserver for the system resolver
   lineinfile:
-    dest: /etc/resolvconf.conf
+    dest: /etc/resolv.conf
```

Comment:
This change doesn't look right. name_server_blacklisting is defined in resolvconf.conf(5) rather than resolv.conf(5)